lib/msal-browser/src/naa/BridgeProxy.ts

```diff
@@ -119,49 +117,35 @@ export class BridgeProxy implements IBridgeProxy {
      * @param request A token request
      * @returns a promise that resolves to a token response or rejects with a BridgeError
      */
-    public getTokenInteractive(request: TokenRequest): Promise<TokenResponse> {
-        return this.sendRequest<TokenResponse>("GetTokenPopup", request);
+    public getTokenInteractive(request: TokenRequest): Promise<AuthResult> {
+        return this.getToken("GetTokenPopup", request);
     }
 
     /**
      * getTokenSilent Attempts to get a token silently from the bridge
      * @param request A token request
      * @returns a promise that resolves to a token response or rejects with a BridgeError
```

Comment:
Do we need to update the description, or will the existing one work? i.e "a promise that resolves to a auth result"